packages/core/src/2d/assembler/SlicedSpriteAssembler.ts

```diff
@@ -23,8 +25,15 @@ export class SlicedSpriteAssembler {
     renderer._subChunk = subChunk;
   }
 
-  static updatePositions(renderer: SpriteRenderer): void {
-    const { width, height, sprite } = renderer;
+  static updatePositions(
+    renderer: SpriteRenderer | SpriteMask | UIImage,
+    width: number,
+    height: number,
+    pivot: Vector2,
+    flipX: boolean = false,
+    flipY: boolean = false
+  ): void {
+    const { sprite } = renderer;
     const { border } = sprite;
```

Comment:
This two lines can be optimized using destructuring to make it more concise and avoid repeated:

const { sprite: { border } } = renderer;

Reply:
Is this a subjective preference?